### ISSUE TITLE
Debian package signing

### ISSUE THREAD

**Issue by prof-milki** [jordansissel/fpm#809]
Just noticed that there is simple RPM signing support, but none yet for DEB packages.

In this case using `dpkg-sig` is certainly simpler than manual package/ar modification.  
Simple patch:

```
  ::Dir.chdir(build_path) do
    safesystem("ar", "-qc", output_path, "debian-binary", "control.tar.gz", datatar)
+    if @attributes[:deb_sign]
+     safesystem("dpkg-sig", "-s", "builder", "-k", @attributes[:deb_sign], output_path)
+   end
```

And an additional flag:

```
+   option "--sign", "KEY", 
+     "Sign the resulting package with named GPG key."
```

(It might be interesting to consolidate it into a global --sign flag. Albeit not sure if that's feasible, since --rpm-sign is a boolean :flag, but the deb version requires a keyname string.)

**Comment by anantshri** [comment]
@jordansissel any chances you are implementing this.

**Comment by neomatrix369** [comment]
Just curious if this feature has been implemented, we already have a `rpm-sign` something for debian would be cool too.

**Comment by mnikhil-git** [comment]
@bernd  @jordansissel  any chance taking a look at this?  :-)

**Comment by Chan9390** [comment]
@jordansissel  Any updates ?

**Comment by jordansissel** [comment]
Indeed. Debian packages can be signed, but i don’t see it much in the wild.
That said, I would support signed deb support and am open to  someone
working on it.
